README.md

```diff
@@ -1,9 +1,11 @@
 * NeXus: https://www.nexusformat.org/
-* citation: [![DOI](https://zenodo.org/badge/19377430.svg)](https://zenodo.org/badge/latestdoi/19377430) (DOI: 10.5281/zenodo.1472392)
-* documentation: https://manual.nexusformat.org/
+* Citation: [![Zenodo DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1472392.svg)](https://doi.org/10.5281/zenodo.1472392)
+* Documentation: https://manual.nexusformat.org/
 * Release Notes: https://github.com/nexusformat/definitions/wiki/Release-Notes
-* build server: http://build.nexusformat.org/
-* travis-ci: syntax check of every NXDL file [![Build Status](https://travis-ci.org/nexusformat/definitions.svg)](https://travis-ci.org/nexusformat/definitions)
+* License: [![License](https://img.shields.io/badge/License-LGPL_v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
+* Continuous Integration: [![Syntax Checking](https://github.com/nexusformat/definitions/actions/workflows/syntax-checks.yml/badge.svg)](https://github.com/nexusformat/definitions/actions/workflows/syntax-checks.yml) [![Travis CI Status](https://travis-ci.org/nexusformat/definitions.svg)](https://travis-ci.org/nexusformat/definitions)
+* Continuous Deployment: [![Publish Documentation](https://github.com/nexusformat/definitions/actions/workflows/publish-sphinx.yml/badge.svg)](https://github.com/nexusformat/definitions/actions/workflows/publish-sphinx.yml)
+* Build Server: http://build.nexusformat.org/
```

Comment:
Not https and do we still want to use this rather than GH actions?

Reply:
So I remove `Build Server: http://build.nexusformat.org/`? And what about `[![Travis CI Status] ...`?

Reply:
Yes, that too, please!

Reply:
Note that https://nexusformat.github.io/definitions (the default GH Actions publishing URL) redirects to https://manual.nexusformat.org/ -- try it!

Reply:
no travis-ci Since #868

Reply:
Others are good, thanks for the contributions!

Reply:
Since we added GH Actions CI workflows, the old CI build server (http://build.nexusformat.org/) is legacy and I suggest we do not refer to it any longer.